bin/gendocs

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -e
 
+# Automatic document generation, validation and helpers
+#
+# bin/gendocs ...
+# 
+# <unspecified> (default)
+#   Generate HTML documentation from JSON schema. Stored in gencode/docs
+#
+# check
+#   Compares the hashes from the last time gendocs was run 
+#
+# feature_bucket [input_file] [output_file]
```

Comment:
gen_bucket?  "feature_bucket" is a thing not an action (like "check")

Reply:
updated